components/list/list-item-drag-handle.js

```diff
@@ -72,6 +72,7 @@ class ListItemDragHandle extends LocalizeCoreElement(FocusMixin(RtlMixin(LitElem
 			:host {
 				display: flex;
 				margin: 0.25rem;
+				pointer-events: auto; /* required since its parent may set point-events: none; (see generic layout) */
```

Comment:
This makes it so `d2l-list-item-drag-handle`'s child `d2l-button-move` is clickable event though its parent sets `pointer-events: none`. Unfortunately this CSS could not be placed in the same spot next to it, since `::slotted()` does not enable targeting elements deeper than their immediate children.